modules/text-input/react/lib/TextInput.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
+import {styled, Themeable} from '@workday/canvas-kit-labs-react-core';
```

Comment:
There's some weird cursor and positioning going on for `TextInput` when RTL. 

First, the cursor stays on the left as you type. If you move the cursor to the end and start typing, the text you type appears at the beginning (far left) of the string. It's like the text is appearing in RTL, but the cursor behavior is still LTR.

Second, the text is sliding slightly as you type. You only notice it on long strings. In one of the `grow` stories, type "asdf" 12-14 times and you will see that the last "f" is partially cut off on the right hand side of the field.

The same behavior is happening on `TextArea`

Reply:
This only happens if you write in LTR language, if you write in an RTL language such as arabic, these issues are not occurring.